crypto/crypto/Cargo.toml

```diff
@@ -58,7 +58,8 @@ assert-private-keys-not-cloneable = []
 cloneable-private-keys = []
 fuzzing = ["proptest", "proptest-derive", "cloneable-private-keys"]
 fiat = ["curve25519-dalek", "ed25519-dalek", "x25519-dalek"]
-vanilla = ["vanilla-curve25519-dalek", "vanilla-ed25519-dalek", "vanilla-x25519-dalek"]
+vanilla = ["vanilla-curve25519-dalek", "vanilla-curve25519-dalek/u64_backend", "vanilla-ed25519-dalek", "vanilla-ed25519-dalek/u64_backend", "vanilla-x25519-dalek", "vanilla-x25519-dalek/u64_backend"]
```

Comment:
1. if you have  `vanilla-(curve|ed|x)25519-dalek/u(64|32)_backend`, then `vanilla-(curve|ed|x)25519-dalek` should be redundant,
2. could we normalize the feature names to `vanilla_u64` and `vanilla_u32`?